### PR TITLE
generate version.txt

### PR DESCRIPTION
**Pull request by dstodolny**
* expose `/version.txt` route with last commit sha code

I tested it locally by building docker image and it works. 🎉

### CODE REVIEW

File: bin/push-new-version.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ echo Authenticating.
 eval $(aws ecr get-login --region=$AWS_REGION)
 
 echo Building docker image
-docker build -t $APP_NAME .
+docker build -t $APP_NAME --build-arg GIT_COMMIT=$(git rev-parse --short HEAD) .
```
Comment (asavin) [inline bin/push-new-version.sh]: nice!

**Comment by gregdorward** [comment]
Unlike with #386 in the Website Honestly project, I can't see the latest version when appending /version.txt to the url. Do I have to access this in a different way @dstodolny?

**Comment by asavin** [comment]
This most likely will not work with Heroku preview app as we're not using Docker for Heroku (i think). A way to test this would be to build a Docker image and run the app from it locally. @dstodolny can probably help.

**Comment by dstodolny** [comment]
This cannot be tested here because the file generation is taking place with `docker build` command. You would have to build your own docker container on your machine to test it.

**Comment by chris-brookes** [comment]
@dstodolny - Can you add the date/time onto this version page also, similar to website-honestly #386

**Comment by gregdorward** [comment]
Happy for this one to be merged based on the other ticket and the fact that this has been checked when run from a docker container. We'll give it another check when deployed to Staging

**Pull request merged**